test/language/expressions/unary-plus/S11.4.6_A3_T3.js

```diff
@@ -13,11 +13,21 @@ if (+"1" !== 1) {
 }
 
 //CHECK#2
-if (isNaN(+"x") !== true) {
-  throw new Test262Error('#2: +"x" === Not-a-Number. Actual: ' + (+"x"));
+if (+new Number("-1") !== -1) {
+  throw new Test262Error('#2: +new String("-1") === -1. Actual: ' + (+new String("-1")));
 }
 
 //CHECK#3
-if (+new Number("-1") !== -1) {
-  throw new Test262Error('#3: +new String("-1") === -1. Actual: ' + (+new String("-1")));
+if (isNaN(+"x") !== true) {
+  throw new Test262Error('#3: +"x" === Not-a-Number. Actual: ' + (+"x"));
+}
```

Comment:
This looks like it's re-ordering the two IfStatements. Is that right? Is it motivated by the missing coverage?

Reply:
Yes, so the `isNaN` cases can be consecutive rather than scattered.